### ISSUE TITLE
[Server] Surface Plugin details in Notification Center

### ISSUE THREAD

**Issue by leecalcote** [meshery/meshery#10509]
### Current Behavior
As a platform with me to extension points, Meshery offers the the ability to enable any different number of plugins. Currently, as a platform Meshery doesn't expose any information into Meshery UI about plugins that have been loaded or have been attempted to load outside of Adapter plugins.

### Desired Behavior
Users are aware of the version, size, and details of the plugins that are loaded.

### Implementation
See https://docs.meshery.io/project/contributing/contributing-server-events

### Acceptance Tests
1. On failure to load plugin, sent this to the Notification Center.
1. On success of loading a plugin, send plugin details (manifest and version) to Notification Center.

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

**Comment by utsavll0** [comment]
Hi Can I take this up?

**Comment by leecalcote** [comment]
Thanks for volunteering, @utsavll0 👍

**Comment by leecalcote** [comment]
@utsavll0 progress?

**Comment by utsavll0** [comment]
I dont have any progress. I understand its an open source effort but I couldnt find any clear documentation on it. Also I asked questions on slack channel and the official forum but couldnt get any good responses. 

Please feel free to assign it to someone else. As I dont see myself working on this issue. Thanks

**Comment by shri771** [comment]
Hi @leecalcote, may I work on this issue?

**Comment by leecalcote** [comment]
Thanks much for jumping in and volunteering. In an effort to help guarantee your success in undertaking assignment of the issue, will you:

1. Confirm that you're able to build the project, specifically, the components necessary to resolve this issue.
1. Confirm the approach you intend to take to resolve the issue.

**Comment by shri771** [comment]
Yes, I can build the Meshery server components with the meshkit repo.

Approach:

I'll add event emission in the adapter loading handlers following the server events contributing guide.

On success: Emit informational event with adapter name, version, and details
On failure: Emit error event with failure details

This covers both success and failure cases. @leecalcote

**Comment by leecalcote** [comment]
@shri771, be aware that adapter capabilities are refreshed with each reload of Meshery UI. You'll want to experiment with this behavior as you work toward support for these events. Please also see the Settings panel in Meshery UI.

**Comment by shri771** [comment]
Thanks for the heads up! I'll experiment with the adapter reload behavior and explore the Settings panel in the UI to understand the current flow better before implementing the events.

**Comment by suzy-g38** [comment]
@leecalcote if nobody is working on this issue, can I work on this?

**Comment by ishwar170695** [comment]
Hi @suzy-g38, just following up on this issue. Are you still planning to work on it?

**Comment by suzy-g38** [comment]
@ishwar170695 you can start working on this.

**Comment by emphor11** [comment]
@ishwar170695 I would like to work on this issue

**Comment by ishwar170695** [comment]
> [@ishwar170695](https://github.com/ishwar170695) I would like to work on this issue

go ahead.
